lighthouse-core/test/audits/first-meaningful-paint-test.js

```diff
@@ -15,34 +15,59 @@
  */
 'use strict';
 
-const Audit = require('../../audits/first-meaningful-paint.js');
+const FMPAudit = require('../../audits/first-meaningful-paint.js');
+const Audit = require('../../audits/audit.js');
 const assert = require('assert');
 const traceEvents = require('../fixtures/traces/progressive-app.json');
+const badNavStartTrace = require('../fixtures/traces/bad-nav-start-ts.json');
 
 /* eslint-env mocha */
 describe('Performance: first-meaningful-paint audit', () => {
   it('scores a -1 when no trace data is present', () => {
-    return Audit.audit({}).then(response => {
-      return assert.equal(response.score, -1);
+    return FMPAudit.audit({}).then(result => {
+      assert.equal(result.score, -1);
+      assert.ok(result.debugString);
     });
   });
 
   it('scores a -1 when faulty trace data is present', () => {
-    return Audit.audit({boo: 'ya'}).then(response => {
```

Comment:
this test was not actually testing faulty trace data after the trace artifact format change :)